### PR TITLE
Add Remmina

### PR DESCRIPTION
**Pull request by mvwieringen**
Needs to be merged after  #5945

### CODE REVIEW

File: components/desktop/remmina/Makefile [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+COMPONENT_VERSION=	1.4.7
+COMPONENT_COMMIT=	da3b4196eabbc5ad9719bd35fa6a633f4aa6337e
+COMPONENT_SUMMARY=	Remmina remote desktop client
+COMPONENT_PROJECT_URL=	https://www.freerdp.com
+COMPONENT_FMRI=		desktop/remote-desktop/remmina
+COMPONENT_CLASSIFICATION=	Applications/Internet
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_COMMIT)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).zip
+COMPONENT_ARCHIVE_URL=	https://github.com/FreeRDP/Remmina/archive/$(COMPONENT_COMMIT).zip
+COMPONENT_ARCHIVE_HASH=	\
```
Comment (AndWac) [inline components/desktop/remmina/Makefile:29]: There is a tagged release on github for Remmina 1.4.7:
https://github.com/FreeRDP/Remmina/archive/v1.4.7.tar.gz
Please use this in COMPONENT_ARCHIVE_URL (construct with parts of this path replaced by the appropriate COMPONENT_* variables)

File: components/desktop/remmina/Makefile [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+# Copyright 2020 Marco van Wieringen
+#
+
+BUILD_BITS= 64
+BUILD_STYLE= cmake
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		Remmina
+COMPONENT_VERSION=	1.4.7
```
Comment (AndWac) [inline components/desktop/remmina/Makefile:21]: COMPONENT_NAME shall be all lower-case characters.

File: components/desktop/remmina/Makefile (hunk 1 above)
Comment (mvwieringen) [reply components/desktop/remmina/Makefile:29]: Ok done.

File: components/desktop/remmina/Makefile (hunk 2 above)
Comment (mvwieringen) [reply components/desktop/remmina/Makefile:21]: That doesn't work as upstream names their dir Remmina if I change that the whole gmake build end ups throwing errors.

Comment (AndWac) [reply components/desktop/remmina/Makefile:21]: Introduce COMPONENT_SRC_NAME= Remmina
and use this in the definition of COMPONENT_SRC.
For an example look at fonts/jetbrains-mono

File: components/desktop/remmina/pkg5 [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        "library/security/openssl",
+        "SUNWcs",
+        "system/library",
+        "system/library/security/libgcrypt",
+        "x11/library/libx11"
+    ],
+    "fmris": [
+        "desktop/remote-desktop/remmina"
+    ],
+    "name": "Remmina"
```
Comment (AndWac) [inline components/desktop/remmina/pkg5]: You'll need to update this file, too. The name is wrong.

**Review by AndWac — APPROVED** [review]
This works on my OI desktop.

**Pull request merged**